.env.local.example

```diff
@@ -0,0 +1,2 @@
+# Make a copy of this file and rename it to `.env.local` and fill out the appropriate information.
+DB_STRING = 'mongodb+srv://[username]:[password]@[cluster].mongodb.net/'
```

Comment:
I believe there's edge cases where you'll run into potential issues with a space between the `=` with `.ENV` variables.

I'd consider changing this to 

`DB_STRING='mongodb+srv://[username]:[password]@[cluster].mongodb.net/'

Also, I recommend adding a new line.